lib/karafka/base_consumer.rb

```diff
@@ -17,17 +17,17 @@ class BaseController
     class << self
       attr_reader :topic
 
-      # Assigns a topic to a controller and build up proper controller functionalities, so it can
+      # Assigns a topic to a consumer and build up proper consumer functionalities, so it can
```

Comment:
Assigns a topic to a consumer and build up proper consumer functionalities so that it can cooperate with the topic settings